node/cmd/guardiand/adminnodes.go

```diff
@@ -118,17 +108,11 @@ func runListNodes(cmd *cobra.Command, args []string) {
 		{"Moonbeam", vaa.ChainIDMoonbeam},
 		{"Arbitrum", vaa.ChainIDArbitrum},
 		{"Xpla", vaa.ChainIDXpla},
+		{"Injective", vaa.ChainIDInjective},
+		{"Neon", vaa.ChainIDNeon},
 		{"Wormchain", vaa.ChainIDWormchain},
 	}
```

Comment:
What happened to arbitrum and is it ok that some networks were now promoted from testnet?

Reply:
Arbitrum is there at line 109.
I think it's fine to always print everything. The values would just be zero.